docs/USAGE.md

```diff
@@ -35,16 +35,13 @@ The app follows the basic principles of the other openHAB UIs, like Basic UI, an
 
 ## Getting Started
 
-When first installed the app is in "Demo Mode".
-To connect to your own openHAB server, navigate to the Settings menu and uncheck the "Demo Mode" option.
-Normally, after unchecking the Demo Mode, the app will be able to use multicast DNS to auto-detect your openHAB server if it is on the same network.
+On first start the app tries to discover your openHAB server. This will only work on local networks and when the server does not enforce whether authentication nor HTTPS. If it failes, you can click on `Go to settings` and manually enter the server settings.
```

Comment:
- whether/nor -> either/or (or remove the 'not' before 'enforce' and use neither/nor)
- failes -> fails